lib/LibManifestPlugin.js

```diff
@@ -53,6 +53,16 @@ class LibManifestPlugin {
 			},
 			(compilation, callback) => {
 				const moduleGraph = compilation.moduleGraph;
+				let targetPaths = {};
+				compilation.chunks.forEach(chunk => {
+					const targetPath = compilation.getPath(this.options.path, {
+						chunk
+					});
+					if (Object.values(targetPaths).includes(targetPath)) {
+						callback(new Error("each chunk must have unique output path"));
+					}
+					targetPaths[chunk.id] = targetPath;
+				});
```

Comment:
It is a bad solution for perf, just use:
```
// We throw an error when we have multiple chunks and no one file interlation
const needError = compilation.chunks.size > 1 && !this.options.path.includes("\[.+\]");
```

Reply:
thank you for your review, I have made some changes.